### DISCUSSION TITLE
Apply several examples on create-next-app

### DISCUSSION THREAD

**Discussion by Carlosbergx2** [vercel/next.js#16626] [Help] [answered]
HI,
IM trying to create an next app but with multi example, like this

create-next-app --example with-next-i18next with-next-sass with-next-seo next-learn

I got this error

Could not locate an example named "with-next-i18next". Please check your spelling and try again.

but I want multiple examples, is that possible?

thanks in advance
Carlos Vieira

**Comment by jamesmosier** [comment, marked as answer]
Hi @Carlosbergx2. Unfortunately that is not possible currently. You can only select 1 example at a time, as there is no logic to merge the code together.

**Reply by Carlosbergx2** [reply]
thanks
and about this
I got this error

Could not locate an example named "with-next-i18next". Please check your spelling and try again.

**Reply by jamesmosier** [reply]
[It looks like](https://github.com/vercel/next.js/tree/canary/examples/with-next-i18next) that example is now located in the next-i18next repo: https://github.com/isaachinman/next-i18next/tree/master/examples/simple

So that will not work with create-next-app
